src/frontend/app/desk/src/app/globals.css

```diff
@@ -0,0 +1 @@
+@import '../cunningham/cunningham-style.css';
```

Comment:
missing new line?

Reply:
Yes right, I added a css linter to see these errors: [🚨(app-desk) add css linter](https://github.com/numerique-gouv/people/pull/3/commits/7cf3dbf9cf251185b63b00fd86704f85fa0d3d2e)